### DISCUSSION TITLE
Creating a Stack without the UI?

### DISCUSSION THREAD

**Discussion by McTalian** [Shopify/shipit-engine#1296] [Q&A] [answered]
Hello,

I was wondering if there was a method or documentation for creating a stack without using the "Add a stack" button in the UI? Some scriptable way perhaps?

Thanks!

**Comment by casperisfine** [comment, marked as answer]
I you just wish to bulk create stacks on your inatance with a script, you can use `bin/rails runner <your-ruby-script>`.

You need some basic Ruby and Rails knowledge. You can look at the `seeds.rb` script for inspiration: https://github.com/Shopify/shipit-engine/blob/31705cc9bdcde7a4e6ddd88715ce27839d9a3a3d/test/dummy/db/seeds.rb#L29-L35

**Comment by McTalian** [comment]
Perfect thanks!
